content/c/dfw-championship.md

```diff
@@ -11,7 +11,7 @@ toclevel=2
 3 = { path = "mjt1-pas-dfw.jpg", caption = "A fan holding the DFW Championship belt at [PpW Mistrz Jest Tylko Jeden](@/e/ppw/2022-03-12-ppw-mistrz-jest-tylko-jeden.md)"}
 +++
 
-The DFW Championship was the main and only title of [Dream Factory Wrestling](@/o/dfw.md). It was introduced in 2016 at [Tournament of Dreams I](@/e/dfw/2016-06-11-dfw-tournament-of-dreams-1.md) with the title being a prize in the titular tournament.
+The DFW Championship was the main and only title of [Dream Factory Wrestling](@/o/dfw.md). It was introduced in 2016 at [Tournament of Dreams II](@/e/dfw/2016-08-20-dfw-tournament-of-dreams-2.md) with the title being a prize in the titular tournament.
```

Comment:
Changed: if CH didn't see the belt until the main event, then it was the second, finals night and not the first one.